gc/verbose/VerboseHandlerOutput.hpp

```diff
@@ -510,11 +503,10 @@ class MM_VerboseHandlerOutput : public MM_Base
 	
 	virtual	void handleConcurrentStartInternal(J9HookInterface** hook, UDATA eventNum, void* eventData) {}
 	virtual void handleConcurrentEndInternal(J9HookInterface** hook, UDATA eventNum, void* eventData) {};
-	virtual const char *getConcurrentTypeString() { return NULL; }
 	
 	virtual void handleConcurrentGCOpStart(J9HookInterface** hook, uintptr_t eventNum, void* eventData) {}
 
-	void handleGCOPOuterStanzaStart(MM_EnvironmentBase* env, const char *type, uintptr_t contextID, uint64_t duration, bool deltaTimeSuccess);
+	void handleGCOPOuterStanzaStart(MM_EnvironmentBase* env, const char *type, uintptr_t contextID, uint64_t duration, bool deltaTimeSuccess, const char *terminationReason = NULL);
```

Comment:
`terminationReason` should be removed given the changes you made for https://github.com/eclipse/omr/pull/5781#discussion_r597392614 .